install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -33,105 +41,3 @@ backend:
       name: http
 {{- end -}}
 {{- end -}}
-
-
-{{/*
-Generate TLS certificates for Hubble Server and Hubble Relay.
-
-Note: these 2 lines, that are repeated several times below, are a trick to
-ensure the CA certs are generated only once:
-
-    $ca := .ca | default (genCA "hubble-ca.cilium.io" (.Values.hubble.tls.auto.certValidityDuration | int))
-    $_ := set . "ca" $ca
-
-Please, don't try to "simplify" them as without this trick, every generated
-certificate would be signed by a different CA.
```

Comment:
This function is move to `templates/hubble/tls-helm/_helpers.tpl` because it's helm certgen specific.
I also keep the comment back, but change a little bit to match with new context. Could you please take a look @gandro

Reply:
Thank you! I proposed a bit a more in-depth adjustment below.